var/spack/repos/builtin/packages/wget/package.py

```diff
@@ -38,7 +38,15 @@ class Wget(Package):
     version('1.17', 'c4c4727766f24ac716936275014a0536')
     version('1.16', '293a37977c41b5522f781d3a3a078426')
 
-    depends_on("openssl")
+    variant(
+        'ssl',
+        default='openssl',
+        values=('gnutls', 'openssl'),
+        description='Specify SSL backend'
+    )
+
+    depends_on('gnutls',  when='ssl=gnutls')
+    depends_on('openssl', when='ssl=openssl')
```

Comment:
For such a small conflict! Maybe git is smart enough to merge the two :laughing:  

Seriously, if it bothers you I'll remove it after you tried the PR and confirmed it works.